### PR TITLE
Update ATS9440.py to fix the 'ABCD' value mapping

### PR DESCRIPTION
**Pull request by gdeboo**
'ABCD' value mapping should be the bitwise or of 0x00000001 | 0x00000002 | 0x00000004 | 0x00000008, which is 15

### CODE REVIEW

File: qcodes/instrument_drivers/AlazarTech/ATS9440.py [hunk 1]
```diff
@@ -238,7 +238,7 @@ def __init__(self, name: str,
                                         'AD': 8,
                                         'BD': 9,
                                         'CD': 10,
-                                        'ABCD': 11})
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/AlazarTech/ATS9440.py:241]: It looks like the mapping of D is probably also wrong then 7 should be changed to 8 and therefor AB, AD and CD are also wrong, could you check that?

Comment (gdeboo) [reply qcodes/instrument_drivers/AlazarTech/ATS9440.py:241]: Yes, D was also wrong. I added another to commit to fix that.

**Comment by jenshnielsen** [comment]
@gdeboo Thanks for the fix. Could you add a changelog entry as described in the pr template ?

- [ ] Create a file in the docs\changes\newsfragments folder with a short description of the change.

This file should be in the format number.categoryofcontribution. Here the number should either be the number
of the pull request. To get the number of the pull request one must
first the pull request and then subsequently update the number. The category of contribution should be
one of ``breaking``, ``new``, ``improved``, ``new_driver`` ``improved_driver``, ``underthehood``.

**Comment by jenshnielsen** [comment]
bors merge

**Review by jenshnielsen — APPROVED** [review]

**Comment by jenshnielsen** [comment]
bors merge

**Pull request merged**